go/json2/marshal_test.go

```diff
@@ -37,3 +37,19 @@ func TestMarshalPB(t *testing.T) {
 		t.Errorf("MarshalPB(col): %q, want %q", b, want)
 	}
 }
+
+func TestMarshalIndentPB(t *testing.T) {
+	col := &vschemapb.Column{
+		Name: "c1",
+		Type: querypb.Type_VARCHAR,
+	}
+	indent := "  "
+	b, err := MarshalIndentPB(col, indent)
+	if err != nil {
+		t.Fatal(err)
+	}
+	want := "{\n  \"name\": \"c1\",\n  \"type\": \"VARCHAR\"\n}"
+	if string(b) != want {
+		t.Errorf("MarshalIndentPB(col, indent): %q, want %q", b, want)
```

Comment:
prefer `assert.Equal` calls here

Reply:
It's consistent with how another test in this file is written. Want me to change it for both?

Reply:
Done